config/run-services.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# ========================================================================================
+# Container initialization script.
+#========================================
+
+# ensure that initialization script is mounted
+
+if [ -d "/mounted/script/" ]
+then
+
+    if [[ -f "/mounted/script/mock_ta.py" ]]; then
```

Comment:
you can put both in a single line by using double brackets and `&&`
Test: 
`if [[ -d /tmp/ && -f /tmp/file ]]; then echo yes; else echo no; fi`